.github/workflows/rust.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       with:
         name: cargo-semver-checks
         version: 0.42.0
-    - run: cargo semver-checks
+    - run: cargo semver-checks --exclude '*-spec'
```

Comment:
There's some funky stuff with the `soroban-token-spec` crate, because it's technically been published before a long time ago, and the tool is messing up looking it up. After the next stable release this can be removed.